test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionBuilderTests.cs

```diff
@@ -19,7 +20,7 @@ public class HubConnectionBuilderTests
         public void HubConnectionBuiderThrowsIfConnectionFactoryNotConfigured()
         {
             var ex = Assert.Throws<InvalidOperationException>(() => new HubConnectionBuilder().Build());
-            Assert.Equal("Cannot create HubConnection instance. A connection was not configured.", ex.Message);
+            Assert.Equal("Cannot create HubConnection instance. An IConnectionFactory was not configured.", ex.Message);
```

Comment:
nit: `nameof` (in both places actually)

Reply:
I think the test should fail if the message changes.